src/Response/Entity/Pim/Variation.php

```diff
@@ -19,6 +19,8 @@
 
 class Variation extends Entity
 {
+    public const EXCLUSION_TAG = 'findologic-exclude';
```

Comment:
We may want to change that naming a bit.

```suggestion
    public const EXCLUSION_TAG_NAME = 'findologic-exclude';
```